#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_arp-manpage.dpatch by William Vera <billy@billy.com.mx>
##
## DP: Character cleanup for UTF-8 compatibility.

@DPATCH@
diff -urNad nemesis-1.4~/man/nemesis-arp.1 nemesis-1.4/man/nemesis-arp.1
--- nemesis-1.4~/man/nemesis-arp.1	2003-10-31 15:29:36.000000000 -0600
+++ nemesis-1.4/man/nemesis-arp.1	2008-07-02 05:23:51.000000000 -0500
@@ -53,7 +53,7 @@
 .I payload-file
 as the payload when injecting ARP/RARP frames.  The maximum payload size is 
 1472 bytes in order to account for the maximum Ethernet frame size.  Payloads
-can also be read from stdin by specifying '-P -'
+can also be read from stdin by specifying '\-P \-'
 instead of a payload-file.
 .IP "-r reply-enable"
 Enables ARP/RARP replies.
